k8s/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: house-innovations-housnav-web-deployment
+  labels:
+    app: house-innovations-housnav-web
+spec:
+  replicas: 2
```

Comment:
Why is replicas only 2?

Reply:
Two pods for now. That's how Clay had it and for now 2 is fine.